cpp/src/barretenberg/honk/composer/ultra_honk_composer_differential.test.cpp

```diff
@@ -0,0 +1,1056 @@
+// TODO(luke): These tests allow for the comparison of the polynomials produced in Ultra Plonk
```

Comment:
Maybe we should chenge the extension to .currently_unused_cpp, so that there is clear indication that this file is not being used